### DISCUSSION TITLE
Lost access to my GitHub primary email

### DISCUSSION THREAD

**Discussion by mrronchi** [community/community#134466] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have lost access to the primary email for my main GitHub account. Is there a way to change the primary email account if you are not able to log in? I happen to know the password, but the authentication page asks me to verify through my email.

Thanks!

**Comment by davevad93** [comment, marked as answer]
Hi @mrronchi ,

I think you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Device verification on sign-in, unable to access email`, fill in the form and explain your issue.

**Reply by davevad93** [reply]
In the `Account or Organization` field select `Other` because I suppose you're going to open the ticket with this account considering that you have lost access to the main one.

**Reply by mrronchi** [reply]
Thanks so much!

**Reply by davevad93** [reply]
You're welcome @mrronchi , I hope you regain access to your main account!
